.circleci/scripts/linux_test_sanity.sh

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
How come we're updating all the vision models? I thought this issue was specific to vgg. Same question for alexnet, resnet etc..

Reply:
@msaroufim This issue exist for five vision models, that they are now updated in the torchvision/master. You can read more about it [here](https://github.com/pytorch/vision/issues/3767#issuecomment-831825167)